src/wrapper.cc

```diff
@@ -20,6 +21,11 @@ Cronet_RESULT (*_Cronet_Engine_Shutdown)(Cronet_EnginePtr self);
 void (*_Cronet_Engine_Destroy)(Cronet_EnginePtr self);
 Cronet_BufferPtr (*_Cronet_Buffer_Create)(void);
 void (*_Cronet_Buffer_InitWithAlloc)(Cronet_BufferPtr self, uint64_t size);
+int32_t (*_Cronet_UrlResponseInfo_http_status_code_get)(

```

Comment:
We need two integers for major and minor revision in this wrapper.

That way we can look those up in the Dart code and throw an exception that the wrapper needs to be recompiled/redownloaded if people update the package but don't run the script again. (Instead of crashing with symbol not found.)

Reply:
Done. Marked as `1.0`. Bumping major version to 1 as a breaking change is introduced.

Breaking API: `InitCronetApi`